src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,49 @@
+package duke.task;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected LocalDate at;
+
+    /**
+     * Creates a type of task with timing
+     *
+     * @param description detail of the task
+     * @param at          time in yyyy-mm-dd
```

Comment:
It seems there may be an extra tab in the description for this JavaDoc line.

Reply:
> It seems there may be an extra tab in the description for this JavaDoc line.

oh I want  to align with the text for description variable so it's easier to see. Is this against the coding convention?